src/stepper/index.tsx

```diff
@@ -34,33 +33,51 @@ export default class Stepper extends PureComponent <StepperProps, StepperState>
     max: defaultInteger,
     min: -defaultInteger,
     format: value => value,
-    onChange: () => {},
   }
 
   state:StepperState = {
     value: this.props.value,
   }
 
+  whileMouseDown:number
+  mouseDownDelay:number
+
   componentDidMount() {
-    this.update(this.state.value, undefined, false)
+    this.update(this.state.value)
```

Comment:
No need for a componentDidMount, you can initialize the state directly with your Min/Max L.39. That way, it would also be intialized at the right value server side (ComponentDidMount is not called server side)